web/atlas.json

```diff
@@ -2113,11 +2113,11 @@
 {"id": "twwfb5", "submitted_by": "Initial-Box-3708", "name": "Wubby Height", "description": "An emote from twitch streamer PaymoneyWubby. He often laments that despite the modest height he is often told he is surprisingly short in person.", "website": "", "subreddit": "/r/paymoneywubby", "center": [450.5, 1511.5], "path": [[442.5, 1508.5], [457.5, 1508.5], [457.5, 1513.5], [442.5, 1513.5], [442.5, 1508.5]]},
 {"id": "twwfa6", "submitted_by": "Ivsucram", "name": "Capoeirista (Capoeira)", "description": "Art depicting a capoeirista; i.e., a person practicing Caopeira. Capoeira is a popular Brazilian martial arts that merges dancing and fighting movements together.", "website": "https://en.wikipedia.org/wiki/Capoeira", "subreddit": "/r/brasil", "center": [1269.5, 592.5], "path": [[1259.5, 612.5], [1259.5, 614.5], [1265.5, 614.5], [1265.5, 602.5], [1269.5, 602.5], [1276.5, 595.5], [1277.5, 597.5], [1277.5, 591.5], [1282.5, 594.5], [1282.5, 593.5], [1280.5, 591.5], [1286.5, 585.5], [1286.5, 573.5], [1288.5, 570.5], [1287.5, 569.5], [1285.5, 569.5], [1281.5, 574.5], [1280.5, 577.5], [1281.5, 577.5], [1281.5, 583.5], [1276.5, 587.5], [1266.5, 587.5], [1265.5, 586.5], [1261.5, 586.5], [1261.5, 587.5], [1251.5, 587.5], [1251.5, 591.5], [1252.5, 592.5], [1261.5, 592.5], [1261.5, 594.5], [1255.5, 594.5], [1255.5, 595.5], [1253.5, 595.5], [1253.5, 594.5], [1252.5, 594.5], [1254.5, 600.5], [1261.5, 600.5], [1261.5, 611.5], [1259.5, 611.5]]},
 {"id": "twwf6m", "submitted_by": "LaKarotMaskeh", "name": "CPE Lyon", "description": "Famous IT and chimical engineer school from Lyon", "website": "https://www.cpe.fr/", "subreddit": "", "center": [781.5, 1747.5], "path": [[759.5, 1741.5], [801.5, 1742.5], [801.5, 1748.5], [796.5, 1748.5], [796.5, 1755.5], [772.5, 1755.5], [772.5, 1749.5], [758.5, 1748.5], [758.5, 1741.5], [760.5, 1741.5]]},
-{"id": "twwf4i", "submitted_by": "Notx_Roh", "name": "Sportshot (Unturned)", "description": "In Unturned, the Sportshot is one of the game's most infamous weapons.", "website": "https://store.steampowered.com/app/304930/Unturned/", "subreddit": "/r/unturned", "center": [655.5, 589.5], "path": [[649.5, 591.5], [657.5, 591.5], [657.5, 590.5], [658.5, 590.5], [658.5, 589.5], [661.5, 589.5], [661.5, 587.5], [652.5, 587.5],[652.5, 588.5],[649.5, 588.5]]},
+{"id": "twwf4i", "submitted_by": "Notx_Roh", "name": "Sportshot (Unturned)", "description": "In Unturned, the Sportshot is one of the game's most infamous weapons.", "website": "", "subreddit": "", "center": [655.5, 589.5], "path": [[649.5, 591.5], [657.5, 591.5], [657.5, 590.5], [658.5, 590.5], [658.5, 589.5], [661.5, 589.5], [661.5, 587.5], [652.5, 587.5],[652.5, 588.5],[649.5, 588.5]]},

```

Comment:
This is unchanged other than removing the website and subreddit. Is this intended?